packages/instantsearch.js/src/widgets/breadcrumb/__tests__/breadcrumb-test.ts

```diff
@@ -1,22 +0,0 @@
-/**
- * @jest-environment jsdom
- */
-
-import breadcrumb from '../breadcrumb';
-
-describe('breadcrumb()', () => {
```

Comment:
should there be some kind of indication in the breadcrumb widget to why it doesn't have any tests? as for now this would be the only widget without tests. Maybe a markdown that points to the common tests?

Reply:
There is `packages/instantsearch.js/src/widgets/breadcrumb/__tests__/breadcrumb.test.tsx`, I moved the remaining tests to it.